include/dlaf/factorization/qr/internal/get_tfactor_nworkers.h

```diff
@@ -0,0 +1,32 @@
+//
+// Distributed Linear Algebra with Future (DLAF)
+//
+// Copyright (c) 2018-2024, ETH Zurich
+// All rights reserved.
+//
+// Please, refer to the LICENSE file in the root directory.
+// SPDX-License-Identifier: BSD-3-Clause
+//
+#pragma once
+
+#include <algorithm>
+#include <cstddef>
+#include <cstdint>
+
+#include <pika/runtime.hpp>
+
+#include <dlaf/tune.h>
+
+namespace dlaf::factorization::internal {
+
+inline size_t getTFactorNWorkers() noexcept {
```

Comment:
What about this to be snake case?

Reply:
see https://github.com/eth-cscs/DLA-Future/pull/1214/commits/36f179d170396472eadd5bd9b3ad1b2a9b43b8c7